### PR TITLE
Fixup lints, remove excess workflow

### PR DESCRIPTION
**Pull request by markan**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,13 +12,14 @@ source:
 build:
   noarch: python
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```

We have a policy to not add noarch packages. https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Recipe_standards.md#the-build-section

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,13 +12,14 @@ source:
...
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv"
 
 requirements:
   host:
     - pip
     - python >=3.5
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - python
```

If not noarch, remove python pins.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,13 +12,14 @@ source:
...
     - setuptools
+    - wheel
   run:
     - pytest >=3.0
     - python >=3.5
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - python
```

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by cbouss — APPROVED** [review]

**Pull request merged**